presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/AddExchanges.java

```diff
@@ -153,41 +148,29 @@ public PlanNode optimize(PlanNode plan, Session session, Map<Symbol, Type> types
         private final PreferredProperties preferredProperties;
```

Comment:
Is this comment still relevant here as you moved logic to `EnforceRequiredJoinMethods`